### DISCUSSION TITLE
User Account Permissions for accessing ads api

### DISCUSSION THREAD

**Discussion by Vignesh-Siva96** [amzn/ads-advanced-tools-docs#47] [Q&A] [answered]
Greetings, 
I am getting empty array as response from "/v2/profiles" endpoint . The user Account that I am using is a amazon seller central account in which it has two merchant accounts with multiple marketplace profiles. The user account has following permissions
1. In advertising console, view only permission for campaign manager and advertising reports.
2. In seller central, view only permission for campaign manager and Product Ads Performance Reports.
Note: I have also tried all three region urls along with the endpoint and stills getting empty array.

Kindly help on this

**Comment by jerirose** [comment]
~~Hi @Vignesh-Siva96, to enable API access via permissions in the Amazon Ads console, the user account needs to be assigned "view/edit" access.~~ This answer is incorrect. See below.

**Comment by jerirose** [comment, marked as answer]
Hi @Vignesh-Siva96 , to follow up on this, I want to note that we have added additional information about permissions and profiles in the advanced tools center. User accounts with view-only access _can_ retrieve profiles using the `accessLevel` query parameter noted in the [technical specification for `GET /profiles`](https://advertising.amazon.com/API/docs/en-us/reference/2/profiles#/Profiles/listProfiles).

An example of a request using `accessLevel` can also be found in the updated [conceptual documentation for profiles](https://advertising.amazon.com/API/docs/en-us/concepts/authorization/profiles#permissions).

For more information on permissions in the Amazon Ads API, see [Permissions](https://advertising.amazon.com/API/docs/en-us/concepts/permissions).

**Comment by Vignesh-Siva96** [comment]
Hi @jerirose,  That's great to know. Thanks for the info.
